pkgs/development/libraries/microsoft-wil/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, gtest
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "microsoft-wil";
+  version = "1.0.230411.1";
+
+  src = fetchFromGitHub {
+    owner = "Microsoft";
+    repo = "wil";
+    rev = "v${version}";
+    hash = "sha256-5DBJDgwZljfiCFMPlPE9aZ41rKnEVzLUcxRTgxqPWHw=";
+  };
+
+  nativeBuildInputs = [ cmake pkg-config ];
+  buildInputs = [ gtest ];
+
+  cmakeFlags = [
+    "-DWIL_BUILD_PACKAGING=OFF"
+    # Unit tests require downloads via nuget
+    "-DWIL_BUILD_TESTS=OFF"
+  ];
+
+  doCheck = true;
```

Comment:
One disables building tests, the other enables the test phase. That doesn't make sense. Means we can likely also drop gtest.